lib/league_stats.rb

```diff
@@ -1,3 +1,3 @@
-class LeagueStats
-  
+module LeagueStats
```

Comment:
It makes sense that the Stats files are modules. It is a good way of leveraging benefits of OOP and not putting everything in StatTracker